bgpd/bgp_aspath.c

```diff
@@ -2096,3 +2101,141 @@ void aspath_print_all_vty(struct vty *vty)
 				       void *))aspath_show_all_iterator,
 		     vty);
 }
+
+static unsigned int bgp_aggr_aspath_hash_key(void *data)
+{
+	struct aspath *aspath = data;
+
+	return jhash(aspath->str, aspath->str_len, 0xdcba4321);
+}
+
+static bool bgp_aggr_aspath_hash_cmp(const void *arg1, const void *arg2)
```

Comment:
Isn't this aspath_cmp?  Why do we need thsi function 2 times?

Reply:
I added a new function to safeguard the aggregation hash just in case the existing function goes through changes in the future.
I've made the changes to use the existing function.

Reply:
Negatory, we are not copy-n-pasting in this code base.

Reply:
> I'm not going to put cut-n-paste code into the code base.

I've removed the duplicate functions. 
The new changes do not have duplicate functions.
Kindly have a look.